components/resources/brave_components_strings.grd

```diff
@@ -351,6 +351,55 @@
         Ad
       </message>
 
+      <message name="IDS_BRAVE_NEWS_BACK_TO_DASHBOARD" desc="Label for the back to dashboard button">
+        Back to <ph name="BEGIN_BOLD">$1</ph>Dashboard<ph name="END_BOLD">$2</ph>
+      </message>
+      <message name="IDS_BRAVE_NEWS_DISABLED_PLACEHOLDER_HEADER" desc="Header for the brave news disabled placeholder">
+        Turn on Brave News, and never miss a story
+      </message>
+      <message name="IDS_BRAVE_NEWS_DISABLED_PLACEHOLDER_SUBTITLE" desc="Subtitle for the brave news disabled placeholder">
+        Customized news feeds, from leading sources, delivered right to your browser. All 100% private.
+      </message>
+      <message name="IDS_BRAVE_NEWS_DISABLED_PLACEHOLDER_ENABLE_BUTTON" desc="Label for the button to reenable Brave News from the disabled placeholder">
+        Turn on Brave News
+      </message>
+      <message name="IDS_BRAVE_NEWS_SEARCH_PLACEHOLDER_LABEL" desc="Label that displays in the searchbox when the user hasn't entered anything">
+        Search for news, site, topic or RSS feed
+      </message>
+      <message name="IDS_BRAVE_NEWS_BROWSE_BY_CATEGORY_HEADER" desc="Header for the browse by category section">
+        Browse by category
+      </message>
+      <message name="IDS_BRAVE_NEWS_LOAD_MORE_CATEGORIES_BUTTON" desc="Label for the button to load more categories">
+        Load more
+      </message>
+      <message name="IDS_BRAVE_NEWS_ALL_SOURCES_HEADER" desc="Header for the all sources section">
+        All Sources
+      </message>
+      <message name="IDS_BRAVE_NEWS_FEEDS_HEADING" desc="The heading for the feeds contianer">
+        Following
+      </message>
+      <message name="IDS_BRAVE_NEWS_SOURCE_COUNT" desc="Label saying how many sources you currently have">
+        <ph name="FEED_COUNT">$1<ex>10</ex></ph> sources
```

Comment:
If this starts to look strange (e.g. in English when there is only 1 source), there is a facility to do differing translation variants for 0, 1 or more counts.

Reply:
Done.